### PR TITLE
249/call to action

### PR DESCRIPTION
**Pull request by VinceXIV**
# Description
Fixed issue #249 
Did the following to achieve this
1. Created a new component named CallToActionComponent
2. copied all the stuff in the original component to this one
3. Modified the HomePageJobsComponent to reuse this newly created component

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# Screenshot (optional)
### text-align: left
Screenshot from 2023-02-25 11-21-51

### text-align: center
Screenshot from 2023-02-25 11-22-13

# How Has This Been Tested?
Tried passing different values to the component at home page and seeing how it affects what is rendered. In the screenshot, what I changed was the text align property to see what it does to what is rendered. The others can be changed too

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by VinceXIV** [comment]
Hello @IanOdhiambo9

File: libs/pages/elewa/home/src/lib/components/home-jobs-section/home-jobs-section.component.html [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
-</section>
\ No newline at end of file
+<elewa-group-call-to-action
+        [message] = "'Become part of the family'"
+        [callToActionIntroducer] = "'View'"
+        [callToAction] = "'open opportunities'"
+        [callToActionURL] = "'jobs'"
+        [align] = "'left'"
+    >
+</elewa-group-call-to-action>
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/home/src/lib/components/home-jobs-section/home-jobs-section.component.html:8]: This should be moved to a shared lib like features/components/banners

File: libs/pages/elewa/home/src/lib/features-elewa-home.module.ts [hunk 2]
```diff
@@ -16,14 +16,16 @@ import { PrevDirective } from './directives/prev.directive';
...
 
 @NgModule({
   imports: [
     CommonModule,
     LayoutModule,
     NgOptimizedImage,
     CardsModule,
-    HomeRoutingModule
+    HomeRoutingModule,
+    SocialImpactModule
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/home/src/lib/features-elewa-home.module.ts:28]: Not allowed, page modules should not import each other. May cause circular dependency later on. Use shared libs.

File: libs/pages/elewa/social-impact/src/lib/components/call-to-action/call-to-action.component.html [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<section id="home-jobs-section-container">
+    <div class="home-jobs-section">
+        <p class="align-{{align}}">{{message}}</p>
+        <p class="align-{{align}}">
+            {{callToActionIntroducer}}
+            <a class="homeLink" routerLink="/{{callToActionURL}}">{{callToAction}}</a>
+        </p>
+    </div>
+</section>
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/social-impact/src/lib/components/call-to-action/call-to-action.component.html:9]: Needs to be in a shared lib should be named elewa-group-call-to-action

**Review by IanOdhiambo9 — CHANGES_REQUESTED** [review]

**Comment by VinceXIV** [comment]
Hey @IanOdhiambo9

**Comment by IanOdhiambo9** [comment]
This issue has been closed.

**Pull request closed without merging**